Lifestyle/fortune.5m.sh

```diff
@@ -0,0 +1,30 @@
+
+fortune="/usr/local/bin/fortune"
+
+echo "|dropdown=false image=$cookie"
+echo "---"
+
+if [[ -x $fortune ]]; then
+  echo "Next cookie|refresh=true image=$cookie"
+  echo "---"
+  echo " |trim=false"
+  echo "$($fortune|/usr/bin/sed 's/-/–/g; s/^/    /; s/$/  |font=Avenir size=14 trim=false/')"
```

Comment:
changing this line to the following worked for me:

```sh
$fortune|/usr/bin/sed 's/-/–/g; s/^/    /; s/$/  |font=Avenir size=14 trim=false/'
```

This passes all tests.  if it doesn't work can you tell me what you are getting/seeing?